apps/typegpu-docs/src/content/examples/tests/tests/index.ts

```diff
@@ -0,0 +1,48 @@
+import tgpu from 'typegpu';
+import * as d from 'typegpu/data';
+import { booleanTests } from './boolean';
+
+const root = await tgpu.init();
+
+const resultBuffer = root.createBuffer(d.i32, 0).$usage('storage');
+const result = resultBuffer.as('mutable');
+
+const computeRunTests = tgpu['~unstable']
+  .computeFn({ in: { num: d.builtin.numWorkgroups }, workgroupSize: [1] })
+  .does(() => {
+    let s = true;
+    s = s && booleanTests();
+
+    if (s) {
+      result.value = 1;
+    } else {
+      result.value = 0;
+    }
+  });
+
+const pipeline = root['~unstable']
+  .withCompute(computeRunTests)
+  .createPipeline();
+
+async function runTests() {
+  pipeline.dispatchWorkgroups(1);
+  return await resultBuffer.read();
```

Comment:
To adapt with the 'createMutable' changes.

```suggestion
  return await resultBuffer.buffer.read();
```